opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -1703,21 +1704,23 @@ namespace Opm {
                             }
                             const auto& wp = i->second;
                             const auto production_controls = wp.productionControls(summaryState);
```

Comment:
Not sure why `summaryState` is here. It is possible something WIP.

Reply:
> Not sure why `summaryState` is here. It is possible something WIP.

It was introduced in commit 511645d12, as part of PR #1831. It's work in progress in the sense that the ACTIONX/UDQ feature is as yet incomplete, but the simulator side of the work is apparently very close to completion (see PR #1865).

Reply:
Thanks for the explanation.